services/content-api.js

```diff
@@ -172,15 +173,15 @@ function getFundingProgramme({ locale, slug, previewMode = false }) {
     }).then(response => get('data.attributes')(response));
 }
 
-function getResearch({ locale, slug = null, searchQuery = null, previewMode = null }) {
+function getResearch({ locale, slug = null, previewMode = null }) {
     if (slug) {
         return fetch(`/v1/${locale}/research/${slug}`, {
             qs: addPreviewParams(previewMode)
-        }).then(response => get('data.attributes')(response));
+        }).then(getAttrs);
     } else {
-        const baseUrl = `/v1/${locale}/research`;
-        const url = searchQuery ? `${baseUrl}?q=${searchQuery}` : baseUrl;
```

Comment:
The search query parameter wasn't being used at all, we _may_ re-add this when we have enough insights articles to warrant it but at the current rate of publishing that's not going to be a problem for a while.